system-variables.md

```diff
@@ -429,6 +429,14 @@ This variable is an alias for `last_insert_id`.
 - Default value: `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
 - This variable controls a number of MySQL compatibility behaviors. See [SQL Mode](/sql-mode.md) for more information.
 
+### sql_require_primary_key <span class="version-mark">New in TBD</span>
+
+- Scope: SESSION | GLOBAL
+- Default value: "OFF"
+- This variable, when turned "ON" will prevent table creation without primary keys in a cluster. Behaves similar to sysvar
+  [sql_require_primary_key](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_sql_require_primary_key) in MySQL 8.0
+- Can be useful with TiCDC based replication, as TiCDC silently ignores tables without primary keys atm
```

Comment:
Not sure if this is the best place for this. Maybe this should (also) be on the TiCDC pages

Reply:
Do you think, this should just be on TiCDC page? or here as well? I just wanted to mention it because as TiCDC silently ignoring tables without PK can come as a surprise, so I think we should include this piece somewhere to let users know that there's a way to prevent that from happening.

Reply:
I think it should be here + on TiCDC. Please also use stronger language:
```suggestion
- It is strongly recommended to enable this variable when using TiCDC. This is because replicating changes to a MySQL sink requires that tables have a primary key.
```